indico/core/storage/backend.py

```diff
@@ -106,6 +106,20 @@ def _ensure_fileobj(self, fileobj):
         """Ensures that fileobj is a file-like object and not a string"""
         return BytesIO(fileobj) if not hasattr(fileobj, 'read') else fileobj
 
+    def _copy_file(self, source, target, chunk_size=1024 * 1024):
```

Comment:
I think that's one of the few places where I would omit the space around `*` - having spaces in the default value of a kwarg always looks pretty odd to me, especially without `()` around it...